src/Storage/Storage.Management/help/Get-AzRmStorageShare.md

```diff
@@ -12,21 +12,33 @@ Gets or lists Storage file shares.
 
 ## SYNTAX
 
-### AccountName (Default)
+### AccountNameSingle (Default)
 ```
 Get-AzRmStorageShare [-ResourceGroupName] <String> [-StorageAccountName] <String> [-Name <String>]
+ [-GetShareUsage] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### AccountName
+```
+Get-AzRmStorageShare [-ResourceGroupName] <String> [-StorageAccountName] <String>
  [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
 ```
 
-### AccountObject
+### AccountObjectSingle
 ```
-Get-AzRmStorageShare -StorageAccount <PSStorageAccount> [-Name <String>]
+Get-AzRmStorageShare -StorageAccount <PSStorageAccount> -Name <String> [-GetShareUsage]
  [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
 ```
 
+### AccountObject
+```
+Get-AzRmStorageShare -StorageAccount <PSStorageAccount> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
 ### ShareResourceId
 ```
-Get-AzRmStorageShare [-ResourceId] <String> [-Name <String>] [-DefaultProfile <IAzureContextContainer>]
+Get-AzRmStorageShare [-ResourceId] <String> [-GetShareUsage] [-DefaultProfile <IAzureContextContainer>]
```

Comment:
@blueww

Reply:
@wyunchi-ms 
Thanks for the catch!
Actually, parameterset "ShareResourceId" should not container "-Name", since resource ID already contains share name information, and my code  always get share name from resource ID when use parameterset "ShareResourceId", "-Name" will be omit.
So it should be a bug to include "-Name" in parameterset "ShareResourceId".
Anyway, to avoid breaking change, I have add "-Name" back to  parameterset "ShareResourceId", and give a warning that -Name will be omit.

Do you have any suggestion how to display breaking change notice only when  use  "-Name" in  parameterset "ShareResourceId"?

Reply:
I think adding the attribute `CmdletParameterBreakingChange("Name", deprecateByVersion="3.0.0", ChangeDescription="Name info is already incloud by ShareResourceId, so it's not necessary any more.")` is OK. What do you think?

Reply:
If use CmdletParameterBreakingChange, the breaking change notice will be show when user -Name, that might be too bother. I only want to show it when user use both -ResourceID and -Name.

If there any way to raise the breaking notice in cmdlet execution code (So I can raise the breaking notice after check user input both -ResourceID and -Name), not by CmdletParameterBreakingChange/CmdletOutputBreakingChange/GenericBreakingChange